src/web/components/SharingPermission/ParticipantsTable.spec.tsx

```diff
@@ -12,12 +12,12 @@ describe('ParticipantsTable', () => {
     expect(screen.getByText('Participant 2')).toBeInTheDocument();
   });
 
-  it('calls onSelectedChange when a participant is clicked', () => {
+  it('calls onSelectedChange when a participant is selected', () => {
     const mockOnSelectedChange = jest.fn();
     render(<Default onSelectedChange={mockOnSelectedChange} />);
 
-    const firstParticipantItem = screen.getByText('Participant 1');
-    fireEvent.click(firstParticipantItem);
+    const firstCheckbox = screen.getAllByRole('checkbox')[0];
```

Comment:
The row in the table no longer triggers the `onClick`, the user must click the checkbox instead. Here I am just grabbing the first one, which in this case will be the 'Select all' checkbox.